Basic_viewer/doc/Basic_viewer/CGAL/Graphics_scene_options.h

```diff
@@ -0,0 +1,101 @@
+
+namespace CGAL {
+
+/*!
+\ingroup PkgBasicViewerClasses
+
+The class `Graphics_scene_options` is used to tune the way that the cells of a given data structure of \cgal are considered.
+The different `std::function` can be modified to change for example the behavior of the drawing.
+`VolumeDescriptor` can be `void` for data structures that do not represent volumes. In such a case, all methods about volumes do not exist.
```

Comment:
Instead of "do not exist"  we might use "are no-ops".   @albert-github @JacksonCampolattaro  ??

Reply:
this issue has not yet been address

Reply:
This is a question, not an issue...

Reply:
@lrineau what expression would you choose?